index.html

```diff
@@ -64,8 +64,9 @@ <h1 class="grow-2" id="logo">Giphy Search</h1>
                 <input class="grow-2 sticky" type="search" name="search" id="search" placeholder="Search GIF here">
                 <button id="search-btn" class="primary-btn"><img src="https://giphy.com/static/img/search-icon.svg"
                         width="30"></button>
-                <button id="toggle" class="primary-btn onClick="toggleLightDark()"><img id="toggle-image" src="./assets/images/lightLamp.svg" alt="https://www.flaticon.com/free-icons/electronics"
+                <button id="toggle" class="primary-btn" onClick="toggleLightDark()"><img id="toggle-image" src="./assets/images/lightLamp.svg" alt="https://www.flaticon.com/free-icons/electronics"
```

Comment:
great! thanks

Reply:
Thanks you for giving me the oportunity:)!